### PR TITLE
Add Grace candidate

### PR DESCRIPTION
**Pull request by rainboyan**
Grace is a forked of Grails 5.1.x, started development in early 2022. An open-source and powerful One Person web application Framework to help developers quickly build Spring Boot applications written in the Groovy programing language. 

https://github.com/graceframework/grace-framework

Please review and merge, thank you.

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/GraceMigrations.scala [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    order = "001",
+    id = "001_add_grace_candidate",
+    author = "rainboyan"
+  )
+  def migration001(implicit db: MongoDatabase) : Candidate = {
+    Candidate(
+      candidate = CandidateName,
+      name = "Grace Framework",
+      description =
+        "An open-source and powerful One-Person web application Framework to help developers quickly build Spring Boot applications written in the Groovy programing language. Grace is a forked of Grails 5 that started development in early 2022.",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/GraceMigrations.scala:22]: Can we change this copy a little?

```
Grace is a powerful open-source, One-Person web application Framework to help developers build Spring Boot applications rapidly using the Groovy programming language. Grace is a fork of Grails 5 that started development in early 2022.
```

File: src/main/scala/io/sdkman/changelogs/GraceMigrations.scala [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  @ChangeSet(
+    order = "002",
+    id = "002_add_new_grace_versions_and_update_default",
+    author = "rainboyan"
+  )
+  def migration002(implicit db: MongoDatabase): Document = {
+    List(
+      "2022.0.0-RC1",
+      "2022.0.0-RC2",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/GraceMigrations.scala:35]: Can we exclude all the release candidates and milestones, please?

File: src/main/scala/io/sdkman/changelogs/GraceMigrations.scala (hunk 1 above)
Comment (rainboyan) [reply src/main/scala/io/sdkman/changelogs/GraceMigrations.scala:22]: yeah, is it very long?
Delete the last sentence, what do you think?
Any others?

File: src/main/scala/io/sdkman/changelogs/GraceMigrations.scala (hunk 2 above)
Comment (rainboyan) [reply src/main/scala/io/sdkman/changelogs/GraceMigrations.scala:35]: OK

File: src/main/scala/io/sdkman/changelogs/GraceMigrations.scala (hunk 1 above)
Comment (rainboyan) [reply src/main/scala/io/sdkman/changelogs/GraceMigrations.scala]: It's updated now.

**Comment by marc0der** [comment]
All that remains is to run scalafmt, sorry!

**Comment by rainboyan** [comment]
Is it releated to `maxColumn = 80`?

**Comment by rainboyan** [comment]
I hope it can be passed now.

**Review by marc0der — APPROVED** [review]

**Pull request merged**